app/presenters/milestones_presenter.rb

```diff
@@ -2,13 +2,9 @@
 
 # Displays milestones for each of the versions for an object
 class MilestonesPresenter
-  # @param [Hash<String,Hash>] milestones the milestone data
-  # @param [Array<Dor::Services::Client::ObjectVersion::Version>] versions the version tag data
   # @param [Array<Dor::Services::Client::UserVersion::UserVersion>] user_versions the user versions data
   # @param [String] druid the druid of the object
-  def initialize(milestones:, versions:, user_versions:, druid:)
-    @milestones = milestones
-    @versions = versions
+  def initialize(user_versions:, druid:)
     @user_versions = user_versions
```

Comment:
Building on what I wrote above:

```suggestion
  def initialize(user_version:, druid:)
    @user_version = user_version
```